src/coreclr/src/tools/aot/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRunSymbolNodeFactory.cs

```diff
@@ -345,20 +343,13 @@ private ISymbolNode CreateTypeHandleHelper(TypeDesc type)
 
         private ISymbolNode CreateMethodHandleHelper(MethodWithToken method)
         {
-            bool useUnboxingStub = method.Method.IsUnboxingThunk();
```

Comment:
After this is gone, does this file still need `using Internal.JitInterface;`?

Reply:
Yes